frontend/src/views/Files.vue

```diff
@@ -3,10 +3,10 @@
     <header-bar v-if="error || req.type == null" showMenu showLogo />
 
     <breadcrumbs base="/files" />
-
+    <listing />
```

Comment:
Put the listing here, just like the sidebar, the headerbar. Then, the listing will always show in the mainpage, sometimes in the background.